docs/installation.md

```diff
@@ -25,3 +25,24 @@ Dev and production ready assets are available in the [`dist` folder of the repos
 
 - Dev version, including source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.dev.js)
 - Production version, minified, no source maps: [kinto.dev.js](https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.min.js)
+
+
+### Subresource integrity
+
+To make sure that you are using the right code when loading from a CDN, you can use subresource
+integrity with the hash provided in ``kinto.min.sha384`` with your release.
+
+For 1.0.0-rc.1, please use it like the following:
+
+.. code-block:: html
+
+    <script src="https://cdn.rawgit.com/Kinto/kinto.js/1.0.0-rc.1/dist/kinto.min.js"
+            integrity="sha384-2EBP0zFAKrL1idZDEzowmRmkeEFAneGUwGUHKmGSWKtTtHRQPbyOJyCk0xCgT1pC"
+            crossorigin="anonymous">
+    </script>
+
+Where ``2EBP0zFAKrL1idZDEzowmRmkeEFAneGUwGUHKmGSWKtTtHRQPbyOJyCk0xCgT1pC``
+is the value in ``dist/kinto.min.sha384``.
```

Comment:
You can calculate it yourself and yes with this PR you can also get the last version on the github repository along with the correct file.